### ISSUE TITLE
Icon Request: fa-Cloudflare

### ISSUE THREAD

**Issue by cascer1** [FortAwesome/Font-Awesome#5554]
I would like an icon for the Cloudflare logo: https://www.cloudflare.com/logo/

**Comment by Soullivaneuh** [comment]
:+1:

**Comment by Soullivaneuh** [comment]
More information with download link here: https://www.cloudflare.com/logo

**Comment by henriquepicanco** [comment]
+1

**Comment by AshboDev** [comment]
+1 for this!

**Comment by JayHoltslander** [comment]
+1 They just updated their logo too.

**Comment by adamschwartz** [comment]
Here’s a quick SVG submission:

```html
<svg xmlns="http://www.w3.org/2000/svg" viewBox="430 20 360 280">
  <path d="M674.8 224c2-7.3 1.3-14-2.2-18.8-3.2-4.5-8.6-7-15-7.4l-123.2-1.6c-.8 0-1.5-.4-2-1-.3-.6-.4-1.4-.2-2.2.4-1.2 1.6-2 3-2.2l124-1.6c14.8-.7 30.8-12.6 36.4-27.2l7-18.5c.4-.8.5-1.6.3-2.4-8-36-40.4-63-79-63-35.6 0-65.8 23-76.6 54.8-7-5.2-16-8-25.5-7-17.2 1.6-31 15.3-32.6 32.4-.4 4.4 0 8.7 1 12.7-28 .7-50.3 23.5-50.3 51.6 0 2.5.3 5 .6 7.5.2 1.3 1.2 2.2 2.4 2.2h227.2c1.3 0 2.5-1 3-2.2l1.6-6zM714 144.8h-3.4c-.8 0-1.5.7-1.8 1.5L704 163c-2 7.2-1.3 13.8 2.2 18.7 3.2 4.5 8.6 7 15 7.4l26.3 1.7c.8 0 1.5.4 2 1 .3.6.4 1.5.2 2.2-.4 1-1.6 2-3 2l-27.2 1.7c-14.8.7-30.7 12.6-36.3 27.2l-2 5c-.4 1 .3 2 1.4 2h93.8c1 0 2-.7 2.4-1.8 1.6-5.8 2.5-12 2.5-18.2 0-37-30.2-67.2-67.3-67.2"/>
</svg>
```
[View](https://cdn.rawgit.com/adamschwartz/d4ea254e9d41820c3596cc39a136292b/raw/f49b8cc1fa5183dbb7c6a8e96fe676a42971d3fd/cloudflare-cloud-svg.html)

@tagliala A couple of notes:

- Not sure if this is relevant to FontAwesome or not, but Cloudflare’s new branding uses a lowercased **f** in Cloudflare.
- I wasn’t sure whether the submission `viewBox` should be a square or not. If so, I’d be happy to make that the case. It also wasn’t clear from the CONTRIBUTING docs whether or not you wanted submissions for company logos (as opposed to conceptual icon requests) as vectors or what. Please let me know if there would be a better format for submission.

**Comment by adamschwartz** [comment]
@tagliala please see https://github.com/cloudflare/font-awesome-glyph for our submission. Thanks!

**Comment by destroyer998** [comment]
👍

**Comment by adamschwartz** [comment]
Oh and here’s a demo:
https://cdn.rawgit.com/cloudflare/font-awesome-glyph/master/test.html

**Comment by BrainCMS** [comment]
+1

Logo resources: https://www.cloudflare.com/logo.

**Comment by adamschwartz** [comment]
@davegandy @tagliala any update on this?

**Comment by MattIPv4** [comment]
+1

**Comment by ohlookitsderpy** [comment]
+1

**Comment by DetectiveHuman** [comment]
+1

**Comment by MrSheldon** [comment]
+1

**Comment by mitchdagamer** [comment]
+1

**Comment by sawyerclever** [comment]
+1

**Comment by zsoltsandor** [comment]
Definitely +1.

**Comment by bhubbard** [comment]
+1

**Comment by hexandcube** [comment]
How is this still not in FA?

**Comment by tagliala** [comment]
Released in 5.15.0
